ideascube/library/templates/library/specimen_import.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+
+{% block content %}
+
+    <div class="row">
+        <h2>{% trans "Import Book Specimens" %}</h2>
+        {% if form.errors %}
+            <div class="error">{{ form.errors }}</div>
+        {% endif %}
+    </div>
+
+    <div class="row">
```

Comment:
Since I'm creating this new template, I'd rather create it the right new way.

So what's the right new way now with PureCSS to do what `row` used to do?